### DISCUSSION TITLE
Import Settings From Stable

### DISCUSSION THREAD

**Discussion by HideakiAtsuyo** [ppy/osu#17891] [Ideas]
Everythnig is in the title !
We can import everything but not the settings !
Would you be able to implement it for the next update ?

The settings has been splitted but doesn't change that much so I think yes 🤔

**Comment by peppy** [comment]
Which settings specifically are you interested in?

Migrating all settings may be an end goal, but it's not an immediately important task, because in general re-configuring the game doesn't take too long.

**Reply by HideakiAtsuyo** [reply]
> Which settings specifically are you interested in?
> 
> Migrating all settings may be an end goal, but it's not an immediately important task, because in general re-configuring the game doesn't take too long.

Mouse sentivitity, screen size, scaling mode, skin, UI, input & gameplay :)

Yeah true but it's faster to import from stable I suppose but since the other things was added (beatmaps, scores etc..) I thought it was already in programm but no and it would be really good :)

**Reply by HideakiAtsuyo** [reply]
And I have a very fast question too, the leaderboard that was showing up before in game while we was beating our old scores etc.. is it still here ? Didn't found the setting when I configured osu.. Maybe I missed it ?

**Reply by peppy** [reply]
The in-game leaderboard still needs to be implemented (it's been added for multiplayer but not local/solo yet). It will come in the near future.

**Reply by HideakiAtsuyo** [reply]
Thanks for all these answers ! I hope I didn't bothered too much ! Have a good day/night peppy :)
